README.md

```diff
@@ -13,11 +13,11 @@
 
 
 ## Table of Contents
-1. [💡 Philosophy](#💡-philosophy)
-2. [🎯 Features](#🎯-features)
-3. [📖 Documentation](#📖-documentation)
-4. [⚡️ Quickstart](#⚡️-quickstart)
```

Comment:
I'm unsure about removing the "Quickstart" entirely. Although there certainly is some redundancy with the RTD tutorials, many developers will find the GH README first, and the quickstart could serve as showcase of how easy it is to get going with MicroCeph. We possibly should update and trim the quickstart though.

Reply:
As discussed, we'll keep the quickstart by making use of the loop files method.